modules/REST/V2/Languages.php

```diff
@@ -0,0 +1,775 @@
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'is_active'       => array(
+					'description' => __( 'Tells whether the language is active.', 'polylang' ),
+					'type'        => 'boolean',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'home_url'        => array(
+					'description' => __( 'Home URL in this language.', 'polylang' ),
+					'type'        => 'string',
+					'format'      => 'uri',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'search_url'      => array(
+					'description' => __( 'Search URL in this language.', 'polylang' ),
+					'type'        => 'string',
+					'format'      => 'uri',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'page_on_front'   => array(
+					'description' => __( 'Identifier of the page on front in this language.', 'polylang' ),
+					'type'        => 'integer',
+					'minimum'     => 0,
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'page_for_posts'  => array(
+					'description' => __( 'Identifier of the page for posts in this language.', 'polylang' ),
+					'type'        => 'integer',
+					'minimum'     => 0,
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'fallbacks'       => array(
+					'description' => __( 'List of language locale fallbacks.', 'polylang' ),
+					'type'        => 'array',
+					'uniqueItems' => true,
+					'items'       => array(
+						'type'    => 'string',
+						'pattern' => '[a-z]{2,3}(?:_[A-Z]{2})?(?:_[a-z0-9]+)?',
+					),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'term_props'      => array(
+					'description' => __( 'Language properties.', 'polylang' ),
+					'type'        => 'object',
+					'properties'  => array(),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'set_default_cat' => array(
+					'description' => __( 'Tells whether the default category must be created when creating a new language.', 'polylang' ),
+					'type'        => 'boolean',
+					'context'     => array( 'edit' ),
+					'default'     => true,
+				),
+			),
+		);
+
+		$taxonomies = $this->translatable_objects->get_taxonomy_names( array( 'language' ) );
+
+		foreach ( $taxonomies as $taxonomy ) {
+			if ( 'language' === $taxonomy ) {
+				$description = __( 'Properties for post types translated by Polylang in this language.', 'polylang' );
+			} elseif ( 'term_language' === $taxonomy ) {
+				$description = __( 'Properties for taxonomy terms in this language.', 'polylang' );
+			} else {
+				/**
+				 * Filters the description to use for language term properties.
+				 *
+				 * @since 3.7
+				 *
+				 * @param string $description The description.
+				 */
+				$description = apply_filters( "pll_{$taxonomy}_properties_description", '' );
+			}
+
```

Comment:
Good idea. Done in [4ea43de](https://github.com/polylang/polylang/pull/1505/commits/4ea43def63cdce464ae2e4771c9716abe1b0eb37).
However, I didn't use the taxonomy description because it doesn't seem fit for this. Instead I created a dedicated class method.